config/initializers/mojfile_uploader_client.rb

```diff
@@ -0,0 +1,6 @@
+require 'mojfile_uploader_api_client'
+
+MojFileUploaderApiClient::HttpClient.configure do |client|
+  client.base_url = ENV.fetch('MOJ_FILE_UPLOADER', 'http://localhost:9292')
```

Comment:
I think we should probably not give a default to fetch - if we miss out on providing this in the environment then that should be an error.

Also, can you add `MOJ_FILE_UPLOADER` to `.env.example` and `.env.test.example` please?